src/pp.js

```diff
@@ -78,6 +78,13 @@ function iterDoc(topDoc, func) {
         for (var i = doc.parts.length - 1; i >= 0; i--) {
           docs.push(doc.parts[i]);
         }
+      } else if (doc.type === "if-break") {
+        if (doc.breakContents !== undefined) {
```

Comment:
If we want to allow a "void" value `null` might be more appropriate. I don't think `ifBreak(undefined, " ")` is great, most people would do `ifBreak(null, " ")`. In that case we should do `if (doc.breakContents != undefined)` instead to also match `null`.